stw.1

```diff
@@ -66,6 +68,9 @@ period between command runs in seconds
 .BI \-A " alpha"
 window background alpha value
 .TP
+.BI \-t " ontop"
```

Comment:
You don't need to specify parameter argument ("ontop"), as, IIUC, `-t` is a *bool* parameter.

Check this:

```
default2 ~ st --help
usage: st [-aiv] [-c class] [-f font] [-g geometry] [-n name] [-o file]
          [-T title] [-t title] [-w windowid] [[-e] command [args ...]]
```

Here `st` has several *value* parameters like `-c` and `-f`. They expect a value to be passed.

Also there are bool parameters like `-a` and `-i` at the very beginning of the usage text. They don't expect a value.

___

Remove "ontop" words from usage string and man page and put `-t` at the beginning right after the program name like in `st` man page.

To grasp this topic a bit better check this:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html